app/controllers/orgs_controller.rb

```diff
@@ -16,25 +16,28 @@ def admin_update
     attrs = org_params
     @org = Org.find(params[:id])
     authorize @org
-    @org.banner_text = params["org_banner_text"]
-    @org.logo = org_params[:logo] if org_params[:logo]
+    @org.logo = attrs[:logo] if attrs[:logo]
+    
+    if attrs[:links].present?
+      @org.links = JSON.parse(attrs[:links])
```

Comment:
I would validate that is an array of objects conforming to { link: string, text: string}